File: Iniciante/Pagina 02/1060/1060.c
#include <stdio.h>
#include <math.h>

int main(){
    double n1, n2, n3, n4, n5, n6;
    int x;
    x = 0;

    scanf("%lf %lf %lf %lf %lf %lf", &n1, &n2, &n3, &n4, &n5, &n6);
    
    if (n1 > 0) {
        (x = x + 1);
    } 
    if (n2 > 0) {
        (x = x + 1);
    } 
    if (n2 > 0) {
        (x = x + 1);
    }
    if (n3 > 0) {
        (x = x + 1);
    } 
    if (n4 > 0) {
        (x = x + 1);
    } 
    if (n5 > 0) {
        (x = x + 1);
    }
    if (n6 > 0) {
        (x = x + 1);
    }
   
    printf("%d valores positivos\n", x);
    
    return 0;
}

File: Iniciante/Pagina 02/1060/1060.py
n1, n2, n3, n4, n5, n6 = input().split()
n1 = float(n1)
n2 = float(n2)
n3 = float(n3)
n4 = float(n4)
n5 = float(n5)
n6 = float(n6)
numbers = [n1, n2, n3, n4, n5, n6]
contador = 0

for x in range(6):
    if numbers[x] > 0:
        contador += 1

print("%d valores positivos\n" %contador)
